<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>QuickDocumentDialog</class>
 <widget class="QDialog" name="QuickDocumentDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>474</height>
   </rect>
  </property>
  <widget class="QWidget" name="layoutWidget" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>430</y>
     <width>461</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <spacer>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>131</width>
        <height>31</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="okButton" >
      <property name="text" >
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton" >
      <property name="text" >
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>461</width>
     <height>406</height>
    </rect>
   </property>
   <layout class="QGridLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item row="5" column="0" >
     <widget class="QLabel" name="labelAuthor" >
      <property name="sizePolicy" >
       <sizepolicy>
        <hsizetype>1</hsizetype>
        <vsizetype>1</vsizetype>
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text" >
       <string>Author</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2" >
     <widget class="QPushButton" name="pushButtonOptions" >
      <property name="text" >
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" >
     <widget class="QComboBox" name="comboBoxPaper" />
    </item>
    <item row="0" column="0" >
     <widget class="QLabel" name="labelClass" >
      <property name="sizePolicy" >
       <sizepolicy>
        <hsizetype>1</hsizetype>
        <vsizetype>1</vsizetype>
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text" >
       <string>Document Class</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1" >
     <widget class="QLineEdit" name="lineEditAuthor" />
    </item>
    <item row="0" column="2" >
     <widget class="QPushButton" name="pushButtonClass" >
      <property name="text" >
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" >
     <widget class="QComboBox" name="comboBoxSize" />
    </item>
    <item row="6" column="0" >
     <widget class="QLabel" name="labelTitle" >
      <property name="sizePolicy" >
       <sizepolicy>
        <hsizetype>1</hsizetype>
        <vsizetype>1</vsizetype>
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text" >
       <string>Title</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" >
     <widget class="QCheckBox" name="checkBoxAMS" >
      <property name="text" >
       <string>AMS Packages</string>
      </property>
      <property name="checked" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="2" >
     <widget class="QPushButton" name="pushButtonEncoding" >
      <property name="text" >
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" >
     <widget class="QComboBox" name="comboBoxClass" />
    </item>
    <item row="1" column="0" >
     <widget class="QLabel" name="labelSize" >
      <property name="sizePolicy" >
       <sizepolicy>
        <hsizetype>1</hsizetype>
        <vsizetype>1</vsizetype>
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text" >
       <string>Typeface Size</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1" >
     <widget class="QLineEdit" name="lineEditTitle" />
    </item>
    <item row="2" column="0" >
     <widget class="QLabel" name="labelPaper" >
      <property name="sizePolicy" >
       <sizepolicy>
        <hsizetype>1</hsizetype>
        <vsizetype>1</vsizetype>
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text" >
       <string>Paper Size</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" >
     <widget class="QLabel" name="labelOptions" >
      <property name="sizePolicy" >
       <sizepolicy>
        <hsizetype>1</hsizetype>
        <vsizetype>1</vsizetype>
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text" >
       <string>Other Options</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1" >
     <widget class="QListWidget" name="listWidgetOptions" />
    </item>
    <item row="3" column="0" >
     <widget class="QLabel" name="labelEncoding" >
      <property name="sizePolicy" >
       <sizepolicy>
        <hsizetype>1</hsizetype>
        <vsizetype>1</vsizetype>
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text" >
       <string>Encoding</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1" >
     <widget class="QCheckBox" name="checkBoxIDX" >
      <property name="text" >
       <string>makeidx Package</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" >
     <widget class="QComboBox" name="comboBoxEncoding" />
    </item>
    <item row="2" column="2" >
     <widget class="QPushButton" name="pushButtonPaper" >
      <property name="text" >
       <string>+</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <pixmapfunction></pixmapfunction>
 <tabstops>
  <tabstop>comboBoxClass</tabstop>
  <tabstop>comboBoxSize</tabstop>
  <tabstop>comboBoxPaper</tabstop>
  <tabstop>comboBoxEncoding</tabstop>
  <tabstop>checkBoxAMS</tabstop>
  <tabstop>checkBoxIDX</tabstop>
  <tabstop>lineEditAuthor</tabstop>
  <tabstop>lineEditTitle</tabstop>
  <tabstop>listWidgetOptions</tabstop>
  <tabstop>pushButtonClass</tabstop>
  <tabstop>pushButtonPaper</tabstop>
  <tabstop>pushButtonEncoding</tabstop>
  <tabstop>pushButtonOptions</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>QuickDocumentDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>278</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>QuickDocumentDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
